packages/wrangler/src/index.ts

```diff
@@ -452,11 +452,6 @@ export function createCLIParser(argv: string[]) {
 	});
 
 	// ai
-	wrangler.command("ai", "🤖 Interact with AI models", (aiYargs) => {
-		return ai(aiYargs.command(subHelp));
-	});
-
-	// constellation
```

Comment:
This looks like maybe some merge conflicts snuck into the PR: this shouldn't remove any `ai`-related functionality